resources/js/src/app/components/myAccount/OrderHistoryItem.js

```diff
@@ -0,0 +1,60 @@
+const ApiService = require("services/ApiService");
+
+Vue.component("order-history-item", {
+
+    delimiters: ["${", "}"],
+
+    props: {
+        template:
+        {
+            type: String,
+            default: "#vue-order-history-item"
+        },
+        orderDetailsTemplate:
+        {
+            type: String,
+            default: "Ceres::Checkout.OrderDetails"
+        },
+        order:
+        {
+            type: Object,
+            default: () =>
+            {}
+        },
+        orderDetailModalId: String
+    },
+
+    data()
+    {
+        return {
+            isLoading: false
+        };
+    },
+
+    created()
+    {
+        this.$options.template = this.template;
+    },
+
+    methods:
+    {
+        setCurrentOrder()
+        {
+            $("#dynamic-twig-content" + this.orderDetailModalId).html("");
```

Comment:
Wenn möglich, bitte über $refs auf Elemente zugreifen, anstatt über IDs

Reply:
Zudem scheint es kein Element mit dieser ID zu geben!

Reply:
Die Datei ist nicht in benutzung und wird gelöscht.